Tests/CustomEntitySearchTest/CustomEntitySearchTest.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <RootNamespace>AzureCognitiveSearch.PowerSkills.Tests.CustomEntitySearchTest</RootNamespace>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.2.0" />
+    <PackageReference Include="MSTest.TestAdapter" Version="1.3.2" />
+    <PackageReference Include="MSTest.TestFramework" Version="1.3.2" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="../../*.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="../../Text\CustomEntitySearch\Entity.cs" />
+  </ItemGroup>
```

Comment:
Usually you add a reference to the entire CSProj, eg:
    <ProjectReference Include="..\..\Text\CustomEntitySearch\CustomEntitySearch.csproj" />

Reply:
Yes, to be clear, the reason why I made a file include for the utility classes was so each project can be as self-contained as possible while still using common logic. In this case, it's entirely different though, and it should be a `ProjectReference`.

Reply:
Would that cause any dependency issues when running CustomEntitySearch?